src/System.Collections/tests/BitArray/BitArray_GetSetTests.cs

```diff
@@ -340,7 +343,7 @@ public static void CopyTo_Size_Invalid<T>(T def, int bits, int arraySize, int in
             AssertExtensions.Throws<ArgumentOutOfRangeException>("index", () => bitArray.CopyTo(array, -1));
             if (def is int)
             {
-                AssertExtensions.Throws<ArgumentException>("destinationArray", string.Empty, () => bitArray.CopyTo(array, index));
+                AssertExtensions.Throws<ArgumentException>(null, string.Empty, () => bitArray.CopyTo(array, index));
```

Comment:
Is this considered a breaking change? Previously, this exception was coming from Array.Copy which returned destinationArray for invalid argument exception. Span.CopyTo does not return a parameter name.

Reply:
ArgumentException is supposed to indicate "you should fix your program", so no one _should_ be depending on it.  But I don't know where we're landing on that in practice right now.  (Of course, this test acknowledges that we throw a different name (string.Empty) on netfx, so it's probably fine)

Reply:
Reverted back to using Array.Copy instead of Span.CopyTo, so no longer need to update this test.